osu.Game/Online/Chat/ExternalLinkOpener.cs

```diff
@@ -44,26 +45,26 @@ public partial class ExternalLinkDialog : PopupDialog
         {
             public ExternalLinkDialog(string url, Action openExternalLinkAction, Action copyExternalLinkAction)
             {
-                HeaderText = "Just checking...";
-                BodyText = $"You are about to leave osu! and open the following link in a web browser:\n\n{url}";
+                HeaderText = "You are leaving osu!";
```

Comment:
Didn't notice this "leave osu!" text as much before (because it was in the body) but it reads like "exiting osu!". Should probably just use:

https://github.com/ppy/osu/blob/ab2677d913552b5d381c91675d8bf6628853585e/osu.Game/Localisation/DeleteConfirmationDialogStrings.cs#L12-L15

because the new designs seem to incorporate severity. Also move to `CommonStrings`.

Reply:
I did consider this. I guess we can try that for now.